context/context.go

```diff
@@ -112,7 +113,12 @@ type SecurityAlgorithm struct {
 	CipheringOrder []uint8 // slice of security.AlgCipheringXXX
 }
 
-func NewPlmnSupportItem() (item factory.PlmnSupportItem) {
+type PlmnSupportItem struct {
+	PlmnId     models.PlmnId   `yaml:"plmnId"`
+	SNssaiList []models.Snssai `yaml:"snssaiList,omitempty"`
+}
+
+func NewPlmnSupportItem() (item PlmnSupportItem) {
 	item.SNssaiList = make([]models.Snssai, 0, MaxNumOfSlice)
 	return
 }
```

Comment:
I do not see this function being used anywhere in the codebase. Should we delete it?
```suggestion
```